src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -10532,6 +10532,13 @@ float Unit::GetUnitSpellCriticalChance(Unit* victim, SpellInfo const* spellProto
                 // Custom crit by class
                 switch (spellProto->SpellFamilyName)
                 {
+                    case SPELLFAMILY_PRIEST:
+                        // Renewed Hope should affect flash heal
+                        if (spellProto->SpellFamilyFlags[0] & 0x800 && spellProto->SpellIconID == 177)
```

Comment:
Wtf do you want now? We are not even talking about any effects, basepoints or whatever................... just what it does in general... and ofc I use spellwork for those kind of information.